test/_resources/Scenarios/default_scenario.c

```diff
@@ -96,13 +96,13 @@ void Init_Context(void)
     // Create TX_TASK_NUMBER Tx Tasks
     for (uint16_t i = 0; i < TX_TASK_NUMBER - 1; i++)
     {
-        Robus_SendMsg(default_sc.App_1.app->ll_service, (msg_t *)(transmit_msg + (sizeof(msg_t) * i)));
+        //Robus_SendMsg(default_sc.App_1.app->ll_service, (msg_t *)(transmit_msg + (sizeof(msg_t) * i)));
     }
 
     // Create LUOS_TASK_NUMBER Luos Tasks
     for (uint16_t i = 0; i < LUOS_TASK_NUMBER; i++)
     {
-        MsgAlloc_LuosTaskAlloc(default_sc.App_1.app->ll_service, (msg_t *)(transmit_msg + (i) * (sizeof(msg_t) + 1)));
+        //MsgAlloc_LuosTaskAlloc(default_sc.App_1.app->ll_service, (msg_t *)(transmit_msg + (i) * (sizeof(msg_t) + 1)));
```

Comment:
Perhaps you should remove those lines for real if you really want to remove them...